ui/app/adapters/cluster.js

```diff
@@ -143,7 +143,8 @@ export default ApplicationAdapter.extend({
     return this.ajax(url, verb, options);
   },
 
-  mfaValidate({ mfa_request_id, mfa_constraints }) {
+  mfaValidate(mfaRequirement) {
```

Comment:
Previously `mfa_requirement` data was destructured in most places, but the new API service consistently returns keys camel cased. I updated most all variables to be camel cased except methods that interact with the older API calls, such as this `POST` request